test/test_proxy_tensor.py

```diff
@@ -1130,6 +1150,9 @@ def forward(self, _a_1, _b_1, _stride_1):
     _local_scalar_dense = torch.ops.aten._local_scalar_dense.default(_a_1);  _a_1 = None
     _local_scalar_dense_1 = torch.ops.aten._local_scalar_dense.default(_b_1);  _b_1 = None
     _local_scalar_dense_2 = torch.ops.aten._local_scalar_dense.default(_stride_1);  _stride_1 = None
+    ge = _local_scalar_dense >= 0
+    ge_1 = _local_scalar_dense_1 >= 0
+    ge_2 = _local_scalar_dense_2 >= 0
```

Comment:
check_is_size call (not traced directly here! We probably could trace it directly if we really wanted to)